From 1bf2257b10189e111f422bb827530b2e0fcc8358 Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Tue, 9 Jul 2019 17:34:13 +0530
Subject: [PATCH 271/767] octeontx2-af: Update hardware workarounds for 95xx A1
 silicon

commit 65b38b035ef96b01204aad28ac7a8f268b416d72 from
git@git.assembla.com:cavium/WindRiver.linux.git

This patch applies workarounds of 95xx A0 to 95xx A1 silicon.

Change-Id: I28f7c81ff418c9dc6139ba0f235d3a8e153ed10e
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12321
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index a376ab90d892..e15f8bb311f9 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -486,7 +486,7 @@ static inline bool is_rvu_95xx_A0(struct rvu *rvu)
 {
 	struct pci_dev *pdev = rvu->pdev;
 
-	return (pdev->revision == 0x10) &&
+	return ((pdev->revision == 0x10) || (pdev->revision == 0x11)) &&
 		(pdev->subsystem_device == PCI_SUBSYS_DEVID_95XX);
 }
 
-- 
2.31.1

